.circleci/config.yml

```diff
@@ -37,7 +37,7 @@ commands:
       - run:
           name: Install and set up melos
           command: |
-            pub global activate melos 0.4.5
+            flutter pub global activate melos 0.4.5
```

Comment:
If you remove the tuneup installation from YAML as suggested in below comments, it might make sense to add a line here `flutter pub global activate tuneup` so that it's included in CI env if/when we eventually re-integrate the analyze tasks in the build.

Reply:
good idea!